app/controllers/patients_controller.rb

```diff
@@ -412,7 +412,7 @@ def update_fields(patient, params, household, propagation)
     end
 
     # Update the patient with updated values.
-    patient.update(params.require(:patient).permit(params_to_update))
+    patient.update(params.permit(params_to_update))
```

Comment:
I think calling `params.permit(params_to_update)` simply gets you `params[:param_1]`, `params[:param_2]`, etc. whereas `params.require(:patient).permit(params_to_update)` gets you `params[:patient][:param_1]`, `params[:patient][:param_2]`, etc.

so I think we might have to change this to `param[:patient].permit(params_to_update)` but I'm not sure if we can call `.permit` after doing `param[:patient]`, but maybe give this a try?

@holmesie thoughts?

Reply:
You can actually nest `permit` calls (example: `params.permit(person: [ :name, { pets: :name } ])`), take a look at https://apidock.com/rails/ActionController/Parameters/permit for reference

Reply:
awesome, thanks @holmesie 

@sgober : so I think we would be able to do something like: `params.permit(patient: params_to_update)`

Reply:
this works for updating the jurisdiction but none of the other monitoring actions (500 error).  not totally sure how we should proceed here

Reply:
You should make this method support both types of updates, or move it to a new route.

Reply:
I ended up just adding patient to the update status call for jurisdictions.  seemed better than messing around with different cases on the backend